src/plugins/textReplace/index.tsx

```diff
@@ -140,55 +207,141 @@ function TextReplace({ update }: { update: () => void; }) {
         update();
     }
 
+    const [value, setValue] = useState("");
+
+    const previews = useMemo(() => {
+        const previews: JSX.Element[] = [];
+
+        let inputText = value;
+
+        for (const rule of textReplaceRules) {
+            inputText = applyRule(rule, inputText);
+            previews.push(<TextInput editable={false} value={inputText} disabled={!rule.isEnabled} />);
+        }
+
+        return previews;
+    }, [value, textReplaceRules]);
+
     return (
         <>
-            <Forms.FormTitle tag="h5" style={{ marginBottom: 0 }}>IS ENABLED</Forms.FormTitle >
-            <Forms.FormTitle tag="h5" style={{ position: "absolute", right: "30px" }}>IS REGEX</Forms.FormTitle >
+            <Flex style={{ justifyContent: "space-between" }}>
+                <Flex>
+                    <Button
+                        onClick={importRules}
+                    >
+                        Import Rules
+                    </Button>
+                    <Button
+                        onClick={mergeRules}
+                    >
+                        Merge Rules
+                    </Button>
+                    <Button
+                        onClick={exportRules}
+                    >
+                        Export Rules
+                    </Button>
+                    <Button
+                        onClick={newRule}
+                        color={Button.Colors.GREEN}
+                    >
+                        New Rule
```

Comment:
```suggestion
                        Create New Rule
```
i want it longer than others